### DISCUSSION TITLE
Can't get little-backup-box to startup

### DISCUSSION THREAD

**Discussion by dander45** [outdoorbits/little-backup-box#131] [Q&A]
I am new to Raspberry Pi so I apologise for any basic questions.
I am trying to load and run the LBB as it appears to be something I make use of.
Following the instructions I have downloaded Raspberry PI OS Lite (32-BIT) to a new SD card and activated SSH in the Imager
I have loaded this into my Raspberry Pi 4 and it goes through a load of uploading etc finally waiting for me to login to Raspberry Pi
I do this and re-boot, it then suggests I change my password, which I do.
then I run the cmd as in the Readme file 
curl -sSL https://raw.githubusercontent.com/outdoorbits/little-backup-box/main/install-little-backup-box.sh | bash 2> install-error.log
the script starts to run....
asks for default mode, I select NONE
ask to install mejiro I say yes
ask to install comitup I say yes
It then does a lot of lines of script 
It goes through a progress bar at the bottom, no apparent errors

After about 20 mins of code going up the screen it gets to another progress bar across the bottom,
mostly selecting - preparing - unpacking
then creates a number of config files
and a pile of setting up
it says home directory cannot be created - couldn't read it all, it went up too quickly
it then does some downloading
does some more get
and another progress bar at the bottom
then installs mejiro
then another progress bar - quite quick thi sone
more gets
another progress bar
then screen goes blank
and a good number of lines then pass starting with OK
then waits for login 
inpup login and system sits at command line prompt.
no errors visible
not sure if I am supposed to run something now or if the installation has failed - any suggestions
in total this took 50minutes, is that about right?

**Comment by outdoorbits** [comment]
Hi, this took much longer than expected, but may depend on your SD card and your internet connection.
Did you try to open web UI in your browser?
https://yourIP

**Comment by dander45** [comment]
Hi thanks for your reply, when I do this I can see the LBB screen – does this mean it is working? When I plug in HDD or SD nothing appears to happen as far as copying goes.
I will try making changes to the web screen and see if that helps

David


David Anderson
Acoustic Engineer
Hebden Sound Ltd.
172 Wadsley lane,
Sheffield
S6 4EE
07774 782503
[facebook sml]
Visit us on Facebook<http://www.facebook.com/accusound.microphone.systems>
Bridge Microphones and Accusound are trading names of Hebden Sound Ltd.

**Comment by dander45** [comment]
Hi,
Sorry to come back with questions,
I can LBB front page and when I click Source to ext.storage I get a log that says “Files missing!” Do Not unplug!
But there doesn’t appear to be any activity on the HHD or SD devices.
Further down the log i.e. a couple of minutes ago when I powered up, it says “Backup Status               0 of 105    %     comparing….
And earlier it said Files missing: 105 files not synced

Is it running or have I got a problem?

Any help would be great

David.



David Anderson
Acoustic Engineer
Hebden Sound Ltd.
172 Wadsley lane,
Sheffield
S6 4EE
07774 782503
[facebook sml]
Visit us on Facebook<http://www.facebook.com/accusound.microphone.systems>
Bridge Microphones and Accusound are trading names of Hebden Sound Ltd.

**Comment by outdoorbits** [comment]
Hi, sounds good. Before your plugged in your USBs, did you click a backup option? After starting a backup by button you will get all instructions by the Log Monitor or, if connected, via your LCD-screen. Sorry, the screen is not yet documented yet, but it's on my to-do... If you are interested, your need a i2c LCD like this: https://www.berrybase.de/sensoren-module/displays/oled-displays/1.3-128x64-oled-display-modul-einfarbig-40-blau-41-spi/i2c-interface-vertikale-stiftleiste and for jumper wires to connect.

**Comment by outdoorbits** [comment]
Hi again, the script has detected, there are files not copied. Could you please set Log Level to 3 (Settings), try to backup again and send your log? You also could go to Tools, plug in your HDD, click mount USB 1, go to Files, folder storage and try to add a file or folder. Do you have write access in this test?

**Comment by dander45** [comment]
Hi,
I think I am having some issues with the SSD that I am using, it appears to go off line for no reason. I am ordering a better one as this may be some of the problems.
I do think the LCD screen would be a good idea I may order one of those as well if you can let me know the connection details.

It may be a day or so before the SSD arrives and maybe a faster SD card.


David Anderson
Acoustic Engineer
Hebden Sound Ltd.
172 Wadsley lane,
Sheffield
S6 4EE
07774 782503
[facebook sml]
Visit us on Facebook<http://www.facebook.com/accusound.microphone.systems>
Bridge Microphones and Accusound are trading names of Hebden Sound Ltd.

**Comment by outdoorbits** [comment]
Could also be a power problem. Did you check the SSD on a pc?

**Comment by dander45** [comment]
Yes, I get a similar problem on a PC so I am thinking there is a problem with it.

David Anderson
Acoustic Engineer
Hebden Sound Ltd.
172 Wadsley lane,
Sheffield
S6 4EE
07774 782503
[facebook sml]
Visit us on Facebook<http://www.facebook.com/accusound.microphone.systems>
Bridge Microphones and Accusound are trading names of Hebden Sound Ltd.
